opus/import/table_schemas/internal_def_product_types.json

```diff
@@ -44,6 +44,31 @@
         "pi_dict_name": "inventory",
         "definition": "Text files ([volume]_inventory.tab) that list every planet and moon inside the instrument field of view for every observation in a particular volume. Associated labels (*.lbl) describe the contents of the text files."
     },
+    {
+        "pi_dict_context": "OPUS_PRODUCT_TYPE",
+        "pi_dict_name": "rms_index",
+        "definition": "Text files created by the Ring-Moon Systems Node that summarize or augment some metadata for all observations in a particular volume. Associated labels (*.lbl) describe the contents of the text files."
+    },
+    {
+        "pi_dict_context": "OPUS_PRODUCT_TYPE",
+        "pi_dict_name": "hstfiles_index",
+        "definition": "Text files ([volume]_hstfiles.tab) that list the associations between data files in a particular HST volume. Associated labels (*.lbl) describe the contents of the text files."
+    },
+    {
+        "pi_dict_context": "OPUS_PRODUCT_TYPE",
+        "pi_dict_name": "raw_image_index",
+        "definition": "Text files ([volume]_raw_image_index.tab) that list the values of various raw image data in a particular volume. Associated labels (*.lbl) describe the contents of the text files."
+    },
+    {
+        "pi_dict_context": "OPUS_PRODUCT_TYPE",
+        "pi_dict_name": "profile_index",
+        "definition": "Text files ([volume]_profile_index.tab) that [...placeholder]. Associated labels (*.lbl) describe the contents of the text files."
+    },
+    {
+        "pi_dict_context": "OPUS_PRODUCT_TYPE",
+        "pi_dict_name": "supplemental_index",
+        "definition": "Text files ([volume]_supplemental_index.tab) that [...placeholder]. Associated labels (*.lbl) describe the contents of the text files."
```

Comment:
Text files ([volume]_supplemental_index.tab) created by the Ring-Moon Systems Node that augment metadata for all observations in a particular volume. Associated labels (*.lbl) describe the contents of the text files.